src/main/scala/ai/eto/rikai/sql/spark/execution/DescribeModelCommand.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 Rikai authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.eto.rikai.sql.spark.execution
+
+import ai.eto.rikai.sql.model.ModelNotFoundException
+import org.apache.spark.sql.catalyst.expressions.Attribute
+import org.apache.spark.sql.{Row, SparkSession}
+
+case class DescribeModelCommand(name: String) extends ModelCommand {
+
+  override val output: Seq[Attribute] = ModelCommand.output
+
+  override def run(session: SparkSession): Seq[Row] = {
+    catalog(session).getModel(name) match {
+      case Some(model) => Seq(Row(model.name, model.uri))
+      case None =>
+        throw new ModelNotFoundException(s"Model '${name}' not found")
```

Comment:
What will user see if we raise exception here?

Reply:
you can try `spark.sql("DESCRIBE TABLE no_exist")`, it throws an `AnalyzeException` exception as well.